sound/soc/sof/ipc4-topology.c

```diff
@@ -1108,6 +1108,14 @@ static int sof_ipc4_init_audio_fmt(struct snd_sof_dev *sdev,
 	if (available_fmt->num_output_formats && i < available_fmt->num_output_formats)
 		base_config->obs = available_fmt->output_pin_fmts[i].buffer_size;
 
+	/* Update the base configuration from the module manifest */
+	sof_ipc4_update_cpc_from_manifest(sdev, swidget->module_info, base_config);
+	if (!base_config->cpc) {
+		base_config->cpc = base_config->obs * base_config->ibs / 1000;
+		dev_dbg(sdev->dev, "Using calculated (obs*ibs/1000) CPC for %s: %u\n",
```

Comment:
looking at the numbers again, it is likely that obs*ibs/100 gives the correct approximation.